eo-parser/src/test/resources/org/eolang/parser/packs/full-syntax.yaml

```diff
@@ -64,6 +64,9 @@ eo: |
           d.
         e.
           f.
+      ^.
+        ^.
+          c
```

Comment:
@Graur try this:

```
$.
  a.
    b
```

Reply:
@yegor256 This fails too

Reply:
@Graur this is exactly my point: your fix is very "narrow". We can do a more generic fix, which will cover all cases (including mine just suggested above).

Reply:
@yegor256 I agree, but can we use `DOT` for all `head`? I corrected code for your comment as I understood it, check please.

Reply:
@Graur you just did it)

Reply:
@yegor256 Good, let me just add some tests